etcdserver/api/rafthttp/metrics.go

```diff
@@ -71,6 +71,68 @@ var (
 		[]string{"From"},
 	)
 
+	snapshotSend = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "etcd",
+		Subsystem: "network",
+		Name:      "snapshot_send",
+		Help:      "Total number of snapshot sends",
```

Comment:
Operators might not understand what this metric is for. They're more familiar with "snapshot save" and "snapshot restore" commands.  Maybe clarify it a bit more? e.g. "Total number of snapshots send from a member to a client or another member" ?